src/Plugin/SchemaPluginInterface.php

```diff
@@ -14,6 +14,14 @@
  */
 interface SchemaPluginInterface extends PluginInspectionInterface, DerivativeInspectionInterface {
 
+  /**
+   * Set the serverID, required for cache id generation.
+   *
+   * @param string
```

Comment:
Function parameter names do require the name to be specified because the line the docblock is attached to may have multiple variables declarations.

```suggestion
   * @param string $serverId
```

Should make PHPStan happy.